### PR TITLE
Adding information on FileNotFoundError when making bem model

### PR DESCRIPTION
**Pull request by MerlinDumeur**
Adresses #304

### CODE REVIEW

File: scripts/source/02-make_forward.py [hunk 1]
```diff
@@ -87,9 +87,18 @@ def run_forward(subject, session=None):
...
+        bem_model = mne.make_bem_model(subject=fs_subject,
+                                       subjects_dir=fs_subjects_dir,
+                                       ico=4, conductivity=conductivity)
+    except FileNotFoundError as exc:
+        message = ("Could not make BEM model due to a missing file. \n"
+                   "Can be solved by setting recreate_bem=True in the config "
+                   "to force recreation of the BEM model, or by deleting the\n"
+                   f" {config.bids_root}/derivatives/freesurfer/subjects/"
+                   f"sub-{subject}/bem/ folder")
+        raise FileNotFoundError(message) from exc
```
Comment (hoechenberger) [inline scripts/source/02-make_forward.py:100]: Why do you do an explicit exception chaining here? I think we can easily do without it, no?

Comment (MerlinDumeur) [reply scripts/source/02-make_forward.py:100]: If I don't, the exceptions are chained with the message `During handling of the above exception, another exception occurred:`. Currently it is `The above exception was the direct cause of the following exception:` which I thought was better.

Comment (MerlinDumeur) [reply scripts/source/02-make_forward.py:100]: I don't know the best practice though so I should probably remove it?

Comment (hoechenberger) [reply scripts/source/02-make_forward.py:100]: I would remove it for now.

I'm super open towards discussing our entire logging and exception handling system in a separate issue / PR!

Comment (agramfort) [reply scripts/source/02-make_forward.py:100]: what trigged your motivation for this PR? just curious

Comment (MerlinDumeur) [reply scripts/source/02-make_forward.py:100]: It's easy to get a crash if something went wrong when creating the BEM model, and there was no indication on what to do, so I expected newcomers to have trouble with it.
I opened issue #304 related to that, and I proposed to open a PR.

**Review by agramfort — APPROVED** [review]
ok for me !

@hoechenberger merge if happy

File: scripts/source/02-make_forward.py (hunk 1 above)
Comment (hoechenberger) [reply scripts/source/02-make_forward.py:100]: @MerlinDumeur Can you remove the explicit exception chaining for now?

**Review by hoechenberger — CHANGES_REQUESTED** [review]
One comment, oh and can you please add a changelog entry? Thanks!

File: docs/source/changes.md [hunk 2]
```diff
@@ -33,3 +33,5 @@ authors:
 - Correctly handle `eog_channels = None` setting after creation of bipolar EEG
   channels
   ({{ gh(311) }} by {{ authors.hoechenberger }})
+- Added instructions on how to handle `FileNotFoundError` when loading the BEM
+  model in the source pipeline by ({{(gh(304)) by authors.MerlinDumeur }})
```
Comment (hoechenberger) [inline docs/source/changes.md:37]:
```suggestion
  model in the source steps ({{ gh(304) }}  by {{ authors.MerlinDumeur }})
```

**Review by hoechenberger — APPROVED** [review]

**Comment by hoechenberger** [comment]
Thanks @MerlinDumeur!

**Pull request merged**